apps/authentication/api/connection_token.py

```diff
@@ -670,18 +669,17 @@ def release_applet_account(self, *args, **kwargs):
 
 
 class AdminConnectionTokenViewSet(ConnectionTokenViewSet):
+    serializer_classes = {
+        'default': AdminConnectionTokenSerializer,
+    }
 
     def check_permissions(self, request):
         user = request.user
         if not user.is_superuser:
             self.permission_denied(request)
 
     def get_queryset(self):
-        return AdminConnectionToken.objects.all()
+        return AdminConnectionToken.objects.all().filter(user=self.request.user)
 
     def get_permed_account(self, user, asset, account_name, protocol):
-        with tmp_to_org(asset.org):
-            account = asset.accounts.all().active().get(name=account_name)
-            account.actions = ActionChoices.all()
-            account.date_expired = timezone.now() + timezone.timedelta(days=365)
-            return account
+        return AdminConnectionToken.get_user_permed_account(user, asset, account_name, protocol)
```

Comment:
The requested codes have been removed here due to being private and confidential.